src/__tests__/helper.ts

```diff
@@ -143,3 +144,9 @@ export const mineBlockAndSync = async ({ lnds, blockHeight }: { lnds: Array<any>
   }
   await Promise.all(promiseArray)
 }
+
+export const set2FA = async ({ wallet, secret }) => {
```

Comment:
this seems somewhat redundant with `save2fa`

Reply:
Agreed, I'll refactor it.